### DISCUSSION TITLE
How many badges can I get?

### DISCUSSION THREAD

**Discussion by plurrall** [community/community#19101] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, I want new badges for my profile, how many can I get?
And how can I get them?

**Comment by felipezanardi** [comment, marked as answer]
You can get 4 badges (Galaxy Brain, Pull Shark, YOLO, and Public Sponsor)
To know how you can get them, [click here](https://github.com/Schweinepriester/github-profile-achievements).

**Reply by plurrall** [reply]
Do I need to pay for them?

**Reply by felipezanardi** [reply]
> Do I need to pay for them?

No, you just need to pay for the "Public Sponsor" badge
